google/resource_container_cluster.go

```diff
@@ -302,12 +307,17 @@ func resourceContainerClusterCreate(d *schema.ResourceData, meta interface{}) er
 		}
 	}
 
-	if v, ok := d.GetOk("master_version"); ok {
+	if v, ok := d.GetOk("min_master_version"); ok {
 		cluster.InitialClusterVersion = v.(string)
 	}
 
-	if _, ok := d.GetOk("node_version"); ok {
-		return fmt.Errorf("cannot set node_version on create, use master_version instead")
+	if v, ok := d.GetOk("node_version"); ok {
+		// ignore -gke.X suffix for now. if it becomes a problem later, we can fix it.
+		mv := strings.Split(cluster.InitialClusterVersion, "-")[0]
```

Comment:
If `min_master_version` is not specified, then `cluster.InitialClusterVersion` will be empty right? This means `mv` will be an empty string. Is that what we want? Does that mean that if we don't set `min_master_version`, we should also leave `node_version` empty?

Reply:
That's what @paddycarver and I ended up deciding on yesterday but now I can't remember for sure why. I went and changed it because it seems reasonable to just specify the `node_version` (since that's what the nodes will be set to anyway) and let GKE handle the master.